cmd/snap/cmd_quota.go

```diff
@@ -172,12 +176,23 @@ func (x *cmdSetQuota) Execute(args []string) (err error) {
 		// currently support that, so currently all snaps specified here are
 		// just added to the group
 
-		return x.client.EnsureQuota(x.Positional.GroupName, x.Parent, names, 0)
-
+		chgID, err = x.client.EnsureQuota(x.Positional.GroupName, x.Parent, names, 0)
+		if err != nil {
+			return err
+		}
 	default:
 		// should be logically impossible to reach here
 		panic("impossible set of options")
 	}
+
+	if _, err := x.wait(chgID); err != nil {
```

Comment:
nitpick: since `chgID` could be empty here, maybe it's worth adding a comment explaining why this operation is safe in that case too.

Reply:
this comes from the preceeding PR